matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/util/MatrixItem.kt

```diff
@@ -159,4 +159,4 @@ fun RoomMemberSummary.toMatrixItem() = MatrixItem.UserItem(userId, displayName,
 
 fun SenderInfo.toMatrixItem() = MatrixItem.UserItem(userId, disambiguatedDisplayName, avatarUrl)
 
-fun SpaceChildInfo.toMatrixItem() = MatrixItem.RoomItem(childRoomId, name ?: canonicalAlias ?: "", avatarUrl)
+fun SpaceChildInfo.toMatrixItem() = MatrixItem.RoomItem(childRoomId, name ?: canonicalAlias, avatarUrl)
```

Comment:
It's not a new thing, but I'm thinking on it now: maybe we could do a small rework and create a `SpaceItem`, which can render automatically using the AvatarRender as a rounded square image.